claat/proto-renderer/html/templates-tests/stylizedtext_test.go

```diff
@@ -4,9 +4,68 @@ import (
 	"testing"
 
 	"github.com/googlecodelabs/tools/claat/proto-renderer/html"
+	"github.com/googlecodelabs/tools/claat/proto-renderer/testing-utils"
 	"github.com/googlecodelabs/tools/third_party"
 )
 
-func TestStylizedTextTemplate(t *testing.T) {
-	// TODO: Next PR
+func TestRenderStylizedTextTemplateEscaping(t *testing.T) {
+	tests := []*testingUtils.RendererTestingBatch{
+		{
+			testingUtils.NewStylizedTextPlain(`<script>alert("you've been hacked!");</script>!`),
+			"&lt;script&gt;alert(&#34;you&#39;ve been hacked!&#34;);&lt;/script&gt;!",
+			true,
+		},
+		{
+			testingUtils.NewStylizedTextPlain("D@ ?òü ǝ$çâpæ? ^>^ '>__<' {&]"),
+			"D@ ?òü ǝ$çâpæ? ^&gt;^ &#39;&gt;__&lt;&#39; {&amp;]",
+			true,
+		},
+		{
+			testingUtils.NewStylizedTextPlain("<h3>**__Markdown not ![esca](ped)__**</h3>"),
+			"&lt;h3&gt;**__Markdown not ![esca](ped)__**&lt;/h3&gt;",
+			true,
+		},
+	}
+	testingUtils.CanonicalRenderingTestBatch(html.Render, tests, t)
+}
+
+func TestRenderStylizedTextTemplate(t *testing.T) {
+	tests := []*testingUtils.RendererTestingBatch{
+		{
+			nil,
+			"",
+			false,
+		},
+		{
+			&tutorial.StylizedText{},
+			"",
+			true,
+		},
+		{
+			testingUtils.NewStylizedTextPlain(""),
+			"",
+			true,
+		},
+		{
+			testingUtils.NewStylizedTextPlain("hello!"),
+			"hello!",
+			true,
+		},
+		{
+			testingUtils.NewStylizedTextStrong("hello!"),
+			"<strong>hello!</strong>",
+			true,
+		},
+		{
+			testingUtils.NewStylizedTextEmphazied("hello!"),
+			"<em>hello!</em>",
+			true,
+		},
+		{
+			testingUtils.NewStylizedTextStrongAndEmphazied("hello!"),
+			"<strong><em>hello!</em></strong>",
+			true,
+		},
+	}
+	testingUtils.CanonicalRenderingTestBatch(html.Render, tests, t)
 }
```

Comment:
The only issue on this is the following:

Per Travis, Go 1.11.x linter doesn't like how this is used later on the test files: 
`composite literal uses unkeyed fields`
So I'll have to retype the fields every time. 
```
tests := []*testingUtils.RendererTestingBatch{
	{
		{
			In: {},
			In: ...,
			Ok: ...,
		},
		"",
		true,
	},
        // other tests ...
}
```

Which makes sense... 

@cassierecher

Reply:
We could make the linter CI errors optional as well for this

Reply:
Let's not override the linter, adding field names is a good idea